src/js/_enqueues/lib/comment-reply.js

```diff
@@ -189,9 +189,10 @@ window.addComment = ( function( window ) {
 			parentId  = getDataAttribute( replyLink, 'commentid' ),
 			respondId = getDataAttribute( replyLink, 'respondelement'),
 			postId    = getDataAttribute( replyLink, 'postid'),
+			replyTo   = getDataAttribute( replyLink, 'replyto'),
```

Comment:
Let's fix the CS I messed up while changing this block ;)

```suggestion
			replyTo   = getDataAttribute( replyLink, 'replyto' ),
```

(also needed for the lines above, sorry)